selection_tools/print_number_of_events.C

```diff
@@ -0,0 +1,196 @@
+
+double B(vector<double> n_vec){
```

Comment:
`B` is definitely not a good name for a function.

If I understand correctly, it returns a sum of elements in the vector. If that's the case, you could replace it with:

```C++
std::reduce(n_vec.begin(), n_vec.end());
```